src/H5Fprivate.h

```diff
@@ -163,7 +162,6 @@ typedef struct H5F_t H5F_t;
 #define H5F_NULL_FSM_ADDR(F)           (H5F_get_null_fsm_addr(F))
 #define H5F_GET_MIN_DSET_OHDR(F)       (H5F_get_min_dset_ohdr(F))
 #define H5F_SET_MIN_DSET_OHDR(F, V)    (H5F_set_min_dset_ohdr((F), (V)))
-#define H5F_VOL_CLS(F)                 (H5F_get_vol_cls(F))
```

Comment:
Should these be replaced with `H5F_VOL_CONN` macros? They wouldn't be used anywhere at present, but it seems like it would be more consistent.